scripts/wildcard-domain-processor/wildcard-expander.ts

```diff
@@ -210,18 +210,27 @@ export function expandWildcardsInAst(ast: AnyRule, wildcardDomains: WildcardDoma
     switch (ast.category) {
         case RuleCategory.Network:
             return expandWildcardsInNetworkRules(ast as NetworkRule, wildcardDomains);
-        case RuleCategory.Cosmetic:
-            // Expand only element hiding rules and exceptions since these kinds of rules
-            // are supported natively in Safari content blockers.
-            // And other cosmetic rule types, e.g. scriptlets in MV3
-            // https://github.com/AdguardTeam/FiltersRegistry/issues/1063,
-            // should not be expanded as they are to be applied by tswebextension where wildcards are not a problem.
-            if (ast.separator.value === CosmeticRuleSeparator.ElementHiding
-                || ast.separator.value === CosmeticRuleSeparator.ElementHidingException
-            ) {
+        case RuleCategory.Cosmetic: {
+            /**
+             * Expand only element hiding rules and their exceptions since these kinds of rules
+             * are supported natively in Safari content blockers.
```

Comment:
in _all_ Safari content blockers? Or in AdGuard's?

Reply:
any Safari content blocker. that's why it is specified with "natively"